config/routes.rb

```diff
@@ -1,6 +1,30 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
+  resources :tweets do
+    member do
+      get 'stats', to: 'tweets#tweet_stats'
+      post 'create', to: 'tweets#create'
+      put 'update', to: 'tweets#update'
```

Comment:
You don't need to define this two routes here,  `resources :tweets ` does it for you